desktop.css

```diff
@@ -0,0 +1,288 @@
+@media screen and (min-width: 768px) {
+  .topHeader {
+    background: #3e3e3e;
+  }
+
+  .menu-content {
+    display: none;
+  }
+
+  .topHeader ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .topHeader ul li {
+    list-style: none;
+    padding: 10px;
+  }
+
+  .topHeader ul a {
+    padding: 5px;
+  }
+
+  .topHeader ul li a {
+    text-decoration: none;
+    color: white;
+    font-family: "lato", sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+  }
+
+  .topHeader ul li a i {
+    margin-right: 5px;
+  }
+
+  .desktopBar ul li a {
+    text-decoration: none;
+    color: #3e3e3e;
+    font-family: "lato", sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+    transition: color 0.3s, transform 0.3s;
+  }
+
+  .topHeader ul a:hover {
+    color: blue;
+  }
+
+  .topHeader ul li a:hover {
+    color: #ec5242;
+  }
+
+  .desktopBar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
+  .desktopBar .logo {
+    margin-top: 10px;
+  }
+
+  .desktopBar .logo img {
+    width: 80%;
+    height: 50px;
+  }
+
+  .desktopBar ul {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+    padding: 20px;
+  }
+
+  .desktopBar ul li {
+    list-style: none;
+  }
+
+  .desktopBar ul li a:hover {
+    color: #ec5242;
+    transform: scale(1.1);
+  }
+
+  .desktopBar ul li:last-child a {
+    color: #ec5242;
+    font-weight: bold;
+    padding: 5px;
+    border: 4px solid #ec5242;
+    border-radius: 5px;
+  }
+}
+
+/* Main */
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(./resources/Background/Mobilebackground.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 100px 0 0 10px;
+    width: 98%;
+    height: 50vh;
+  }
+
+  .introContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100%;
+  }
+
+  .content {
+    width: 80%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+  }
+
+  .content h1 {
+    font-size: 35px;
+    font-weight: 400;
+  }
+
+  .content h2 {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .content p {
+    font-size: 25px;
+    font-weight: 400;
+  }
+
+  .content h3 {
+    font-size: 30px;
+    font-weight: 700;
+  }
+
+  .content h4 {
+    font-size: 20px;
+    font-weight: 700;
+  }
+}
+
+/* Main Program */
+@media screen and (min-width: 768px) {
+  .program {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, Please ensure that your hover effects and texts align correctly in the design to match the design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/e1f42878-6d95-4953-b4dd-d11f86726755)


</td>
      <td>

![Design](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/39713dd4-1fa1-4d37-a843-c40a474f3270)

</td>
    </tr>
  </table>